### PR TITLE
fix(draggable): pointer event not triggered outside wrapper when chil…

### PR DESCRIPTION
**Pull request by vladimirivanoviliev**
…dren are reordered

this is related to pointCapture

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -107,13 +107,12 @@ export class Draggable {
 
         this._pointerdown = (e) => {
             if (e.isPrimary && e.button === 0) {
-                bind(this._element, "pointermove", this._pointermove);
-                bind(this._element, "pointerup", this._pointerup);
-                bind(this._element, "contextmenu", preventDefault);
+                bind(document, "pointermove", this._pointermove);
+                bind(document, "pointerup", this._pointerup);
+                bind(document, "contextmenu", preventDefault);
```
Comment (tsvetomir) [inline src/main.js]: Is it necessary to move `contextmenu` to the document as well?

File: src/main.js [hunk 2]
```diff
@@ -107,13 +107,12 @@ export class Draggable {
 
         this._pointerdown = (e) => {
             if (e.isPrimary && e.button === 0) {
-                bind(this._element, "pointermove", this._pointermove);
-                bind(this._element, "pointerup", this._pointerup);
-                bind(this._element, "contextmenu", preventDefault);
+                bind(document, "pointermove", this._pointermove);
+                bind(document, "pointerup", this._pointerup);
```
Comment (tsvetomir) [inline src/main.js]: The `pointermove` and `pointerup` events may fire for a different pointer when using multi-touch.
Maybe tying them to the same `pointerId` that triggered `pointerdown` can guard against that?

**Comment by danielkaradachki** [comment]
the change will also fix: https://github.com/telerik/kendo-angular/issues/2148

**Comment by danielkaradachki** [comment]
Tested all packages that use draggable. Besides the fix for the grid I didn't notice any other changes. If there aren't any other suggestions, I say we merge it and I will retest on develop.

**Pull request merged**